src/much-select.js

```diff
@@ -305,6 +311,10 @@ class MuchSelect extends HTMLElement {
       if (oldValue !== newValue) {
         this.isInMultiSelectMode = newValue;
       }
+    } else if (name === "multi-select-single-removable-options") {
```

Comment:
In your example you call the attribute `single-item-removal`... are you going with `multi-select-single-removable-options`?

Reply:
You probably want to add the attribute to the `observedAttributes` too.